docs/assessment.md

```diff
@@ -2,53 +2,140 @@ Migration Assessment Report
+  - [Table counts by storage](#table-counts-by-storage)
+  - [Table counts by schema and format](#table-counts-by-schema-and-format)
+  - [Database Summary](#database-summary)
+  - [External Locations](#external-locations)
+  - [Mount Points](#mount-points)
+  - [Table Types](#table-types)
+  - [Incompatible Clusters](#incompatible-clusters)
+  - [Incompatible Jobs](#incompatible-jobs)
+  - [Incompatible Delta Live Tables](#incompatible-delta-live-tables)
+  - [Incompatible Global Init Scripts](#incompatible-global-init-scripts)
+- [Assessment Finding Index](#assessment-finding-index)
+    - [AF101 - not supported DBR: ##.#.x-scala2.12](#af101---not-supported-dbr-x-scala212)
+    - [AF102 - not supported DBR: ##.#.x-cpu-ml-scala2.12](#af102---not-supported-dbr-x-cpu-ml-scala212)
+    - [AF103 - not supported DBR: ##.#.x-gpu-ml-scala2.12](#af103---not-supported-dbr-x-gpu-ml-scala212)
+    - [AF111 - Uses azure service principal credentials config in cluster.](#af111---uses-azure-service-principal-credentials-config-in-cluster)
+    - [AF112 - Uses azure service principal credentials config in Job cluster.](#af112---uses-azure-service-principal-credentials-config-in-job-cluster)
+    - [AF113 - Uses azure service principal credentials config in pipeline.](#af113---uses-azure-service-principal-credentials-config-in-pipeline)
+    - [AF114 - unsupported config](#af114---unsupported-config)
+    - [AF115 - unsupported config: spark.databricks.passthrough.enabled](#af115---unsupported-config-sparkdatabrickspassthroughenabled)
+    - [AF116 - No isolation shared clusters not supported in UC](#af116---no-isolation-shared-clusters-not-supported-in-uc)
+    - [AF117 - cluster type not supported](#af117---cluster-type-not-supported)
+    - [AF201 - Inplace Sync](#af201---inplace-sync)
+    - [AF202 - Asset Replication Required](#af202---asset-replication-required)
+    - [AF203 - Data in DBFS Root](#af203---data-in-dbfs-root)
+    - [AF204 - Data is in DBFS Mount](#af204---data-is-in-dbfs-mount)
+    - [AF210 - Non-DELTA format: CSV](#af210---non-delta-format-csv)
+    - [AF211 - Non-DELTA format: DELTA](#af211---non-delta-format-delta)
+    - [AF212 - Non-DELTA format](#af212---non-delta-format)
+    - [AF221 - Unsupported Storage Type](#af221---unsupported-storage-type)
+  - [AF300 - AF399](#af300---af399)
+    - [AF300.6 - 3 level namespace](#af3006---3-level-namespace)
+    - [AF300.1 - r language support](#af3001---r-language-support)
+    - [AF300.2 - scala language support](#af3002---scala-language-support)
+    - [AF300.3 - Minimum DBR version](#af3003---minimum-dbr-version)
+    - [AF300.4 - ML Runtime cpu](#af3004---ml-runtime-cpu)
+    - [AF300.5 - ML Runtime gpu](#af3005---ml-runtime-gpu)
+    - [AF301.1 - spark.catalog.x](#af3011---sparkcatalogx)
+    - [AF301.2 - spark.catalog.x](#af3012---sparkcatalogx)
+  - [AF302.x - Arbitrary Java](#af302x---arbitrary-java)
+    - [AF302.1 - Arbitrary Java](#af3021---arbitrary-java)
```

Comment:
Duplicate heading names make no sense for the user. Rewrite all headings to be unique and don't look like a copy-paste